test/services/CapiTest.scala

```diff
@@ -0,0 +1,204 @@
+      override def apiUrl: String = ???
+      override def fields: Option[ContentFields] = Some(new DumbContentFieldsObject(internalPageCode))
+      override def elements: Option[Seq[Element]] = ???
+      override def references: Seq[Reference] = ???
+      override def isExpired: Option[Boolean] = ???
+      override def blocks: Option[Blocks] = ???
+      override def rights: Option[Rights] = ???
+      override def crossword: Option[Crossword] = ???
+      override def stats: Option[ContentStats] = ???
+      override def section: Option[Section] = ???
+      override def debug: Option[Debug] = ???
+      override def isGone: Option[Boolean] = ???
+      override def isHosted: Boolean = ???
+      override def pillarId: Option[String] = ???
+      override def pillarName: Option[String] = ???
+      override def atoms: Option[Atoms] = ???
+    }
+
+    "no internal page code" in {
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(internalPageCode = None))
+      result shouldBe None
+    }
+
+    "some internal page code" in {
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject())
+      result should not be None
+      result.get.internalPageCode shouldBe testInternalPageCode
+    }
+
+    "don't show" in {
+      val tag = DumbTagObject(
+        "tone/sarcastic",
+        com.gu.contentapi.client.model.v1.TagType.Tone)
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(tags = List(tag)))
+      result should not be None
+      result.get.showByline shouldBe false
+      result.get.showQuotedHeadline shouldBe false
+      result.get.imageCutoutReplace shouldBe false
+      result.get.cutout shouldBe None
+    }
+
+    "comment showByline" in {
+      val tag = DumbTagObject(
+        "tone/comment",
+        com.gu.contentapi.client.model.v1.TagType.Tone)
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(tags = List(tag)))
+      result should not be None
+      result.get.showByline shouldBe true
+    }
+
+    "cartoon showByline" in {
+      val tag = DumbTagObject(
+        "type/cartoon",
+        com.gu.contentapi.client.model.v1.TagType.Type)
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(tags = List(tag)))
+      result should not be None
+      result.get.showByline shouldBe true
+    }
+
+    "showQuotedHeadline" in {
+      val tag = DumbTagObject(
+        "tone/comment",
+        com.gu.contentapi.client.model.v1.TagType.Tone)
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(tags = List(tag)))
+      result should not be None
+      result.get.showQuotedHeadline shouldBe true
+    }
+
+    "imageCutoutReplace" in {
+      val tag = DumbTagObject(
+        "tone/comment",
+        com.gu.contentapi.client.model.v1.TagType.Tone)
+      val result = GuardianCapi.prefillMetadata(new DumbContentObject(tags = List(tag)))
+      result should not be None
+      result.get.imageCutoutReplace shouldBe true
```

Comment:
As a general note I prefer the use of `OptionValues` as the error messages are somewhat better. (http://www.scalatest.org/user_guide/other_goodies)